src/extension.ts

```diff
@@ -77,36 +80,23 @@ export function activate(context: vscode.ExtensionContext): void {
 		if (node && node.label) {
 			setStatusLineMessageAndShow(`Removing Apache Camel K Integration...`);
 			let integrationName : string = node.label;
-			if (useProxy) {
-				utils.shareMessage(mainOutputChannel, 'Removing ' + integrationName + ' via Kubernetes Rest Delete');
-				await stopIntegrationViaRest(integrationName).catch( (error) => {
-					utils.shareMessage(mainOutputChannel, `rest error: ${error}`);
-				});
-			} else {
-				utils.shareMessage(mainOutputChannel, 'Removing ' + integrationName + ' via Kamel executable Delete');
-				let commandString = 'kamel delete "' + integrationName + '"';
-				child_process.exec(commandString, (error, stdout, stderr) => {
-					if (error) {
-						utils.shareMessage(mainOutputChannel, `exec error: ${error}`);
-						return;
-					}
-					if (stdout) {
-						// empty for now, but here in case we need it
-					}
-					if (stderr) {
-						console.log(`stderr: ${stderr}`);
-					}
-				});
-				await integrationutils.killChildProcessForIntegration(integrationName).then( (boolResult) => {
-					console.log(`Removed the child process running in the background for ${integrationName}: ${boolResult}`);
-				}).catch( (err) => {
-					console.log(err);
-				});
-				await removeOutputChannelForIntegrationViaKubectl(integrationName)
-				.catch( (err) => {
-					console.log(err);
-				});
-			}
+			let kamelExe = kamel.create();
+			utils.shareMessage(mainOutputChannel, 'Removing ' + integrationName + ' via Kamel executable Delete');
+			let args : string[] = ['delete', `${integrationName}`];
+			await kamelExe.invokeArgs(args)
+				.then( /* empty for now but here in case we need it */ )
```

Comment:
what is the advantage if having an empty then compared to having no then?

Reply:
Leaving it empty without a comment is more confusing than with the comment. And I'd rather have an easy point of entry if we need it.